outputs.tf

```diff
@@ -32,7 +32,7 @@ output "cluster" {
 }
 
 output "velero_storage" {
-  value       = local.config_velero.install ? module.velero[0].s3 : null
+  value       = local.config_velero.install ? module.velero.s3 : null
```

Comment:
This should output if the bucket is created or exists, not if velero is installed. You can prob just do `try(module.velero.s3)`